packages/html/src/index.test.ts

```diff
@@ -31,6 +31,7 @@ describe('html', () => {
 
   it('errors when interpolating object', () => {
     assert.throws(
+      // @ts-expect-error
```

Comment:
Oh nice! I didn't realize we could do that, but it makes sense that we want to.

Reply:
Yeah, `// @ts-expect-error` is awesome. Unlike `// @ts-ignore`, it actually ensures that there *is* an error there, and if there stops being an error at some point in the future, the lack of error itself becomes an error and you're forced (in a good way) to deal with it!